packages/apputils-extension/src/themesplugins.ts

```diff
@@ -113,13 +117,29 @@ export const themesPlugin: JupyterFrontEndPlugin<IThemeManager> = {
       execute: () => manager.toggleThemeScrollbars()
     });
 
+    commands.addCommand(CommandIDs.changeFont, {
+      label: args =>
+        args['enabled'] ? `${args['font']}` : trans.__(`waiting for fonts`),
```

Comment:
Maybe use normal strings and not interpolation ones?

`trans.__(`waiting for fonts`)` -> `trans.__('waiting for fonts')`

Reply:
done